ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/benchmarks/P256VerifyBenchmark.java

```diff
@@ -39,3139 +39,3152 @@ public P256VerifyBenchmark(final PrintStream output, final BenchmarkConfig bench
 
   @Override
   public void runBenchmark(final Boolean attemptNative, final String fork) {
+    final SequencedMap<String, Bytes> testCases = addTestCases();
+
+    final SECP256R1 signatureAlgorithm = new SECP256R1();
+    if (attemptNative != null && (!attemptNative || !signatureAlgorithm.maybeEnableNative())) {
+      signatureAlgorithm.disableNative();
+      P256VerifyPrecompiledContract.disableNativeBoringSSL();
+      ;
+    }
+    output.println(
+        P256VerifyPrecompiledContract.isNativeBoringSSL() ? "Native secp256r1" : "Java secp256r1");
+
+    final P256VerifyPrecompiledContract contract =
+        new P256VerifyPrecompiledContract(gasCalculatorForFork(fork), signatureAlgorithm);
+
+    //    warmIterations = warmIterations / testCases.size();
+    //    execIterations = execIterations / testCases.size();
```

Comment:
There are currently 778 test cases.

Dividing by this means a warmup of 20_000 iterations gets reduced to 25 which isn't sufficient.

It does mean the benchmark takes a long time to run so might need further tweaking.

Reply:
Going to leave this as is, shouldn't be a blocker for this PR.